ifrs17/Import/5ImportScope-ToIfrsVar.ipynb

```diff
@@ -358,9 +366,12 @@
                 "public interface DeferrableAmortizationFactorToIfrsVariable: IScope<ImportIdentity, ImportStorage>",
                 "\n{",
                 "\n    private string EconomicBasis => GetStorage().GetEconomicBasisDriver(Identity.DataNode);",
+                "\n    private bool hasAmortizationStep => GetStorage().DataNodeDataBySystemName[Identity.DataNode].ValuationApproach == ValuationApproaches.PAA",
+                "\n        ? GetScope<DeferrableCashflowToIfrsVariable>(Identity).Deferrable.Where(iv => iv.Values != null).Any(iv => Math.Abs(iv.Values.GetValidElement(Identity.ProjectionPeriod)) > Precision)",
```

Comment:
You can have this repetitive part of the code outside the ternary op:

```
private bool hasAmortizationStep => (GetStorage().DataNodeDataBySystemName[Identity.DataNode].ValuationApproach == ValuationApproaches.PAA"
? GetScope<DeferrableCashflowToIfrsVariable>(Identity).Deferrable
: GetScope<DeferrableActualToIfrsVariable>(Identity).DeferrableActual)
.Where(iv => iv.Values != null).Any(iv...
```

Reply:
Thanks you are right. But in this case the DeferrableActualToIfrsVariable scope will be deleted in the next PR. So for the time being I would leave it as is.